app-text/robodoc/robodoc-4.99.43.ebuild

```diff
@@ -0,0 +1,36 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="Automating Software Documentation"
+HOMEPAGE="http://www.xs4all.nl/~rfsber/Robo/robodoc.html"
+SRC_URI="http://rfsber.home.xs4all.nl/Robo/archives/${P}.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="examples"
+
+RDEPEND=">=dev-util/ctags-5.3.1"
+DEPEND="${RDEPEND}"
+
+src_configure() {
+	econf \
+		--docdir=/usr/share/doc/${PF} \
+		--disable-dependency-tracking
+}
+
+src_install() {
+	emake DESTDIR="${D}" install || die "emake install failed"
```

Comment:
You can replace this line with 'default'. Also, emake doesn't need die in here.

Reply:
emake got replace with default for src_install as noted